server/routers/activityRecord.ts

```diff
@@ -0,0 +1,97 @@
+import { TRPCError } from '@trpc/server';
+import { z } from 'zod';
+
+import { createTRPCRouter, privateProcedure } from '../trpc';
+
+export const activityRecordRouter = createTRPCRouter({
+  list: privateProcedure.query(({ ctx }) =>
+    ctx.prisma.activityRecord.findMany({
+      where: {
+        userId: ctx.user?.id,
+      },
+    }),
+  ),
+
+  create: privateProcedure
+    .input(z.object({ activityId: z.number(), addedAmount: z.number() }))
+    .mutation(async ({ input, ctx }) => {
+      const newActivityRecord = await ctx.prisma.activityRecord.create({
```

Comment:
Do the implicit return as with the above.

Reply:
@GLObus303 No really sure how you mean this one.